src/components/Homepage/LendByCategory/LoanCategoriesSection.vue

```diff
@@ -0,0 +1,275 @@
+<template>
+	<div>
+		<div>
+			<a
```

Comment:
For accessibility these would be better served as `<button>` elements styled to look like links. I'll make an addition to KvButton to handle that. In Chrome at least they aren't keyboard focusable without an href value even with the role=button.